crates/red_knot_python_semantic/resources/mdtest/descriptor_protocol.md

```diff
@@ -22,22 +22,25 @@ class Ten:
         pass
 
 class C:
-    ten = Ten()
+    ten: Ten = Ten()
```

Comment:
This change (and similar ones below) isn't needed, but I didn't want to see `Unknown` everywhere. We have an extra test for the undeclared case further down below.